### DISCUSSION TITLE
Error in running main page

### DISCUSSION THREAD

**Discussion by pavitraag** [code-a2z/jarvis#163] [Q&A] [answered]
i created a issue to create quiz generator...quiz generator is running alone but together wen i try to run as jarvis.py then i got this error
![image](https://github.com/user-attachments/assets/fdad0c78-e5eb-4e43-8596-57403317a769)
Please tell me how to solve this

**Comment by Avdhesh-Varshney** [comment]
Because of confidentiality, I cannot give the `MongoDB URL`. To setup this project on your system, you can create a local mongodb connection from mongodb compass and setup your url in `secrets.toml` file as the naming convention mentioned in `secrets.example.toml` file.

![image](https://github.com/user-attachments/assets/32f9b6a3-3a3f-42a5-8f8b-eec7615c4642)

**Reply by Avdhesh-Varshney** [reply, marked as answer]
After setup your url in secrets.toml file, first you need to go for `signup` and then go for `login`
